src/apps/chem/xcfunctional_libxc.cc

```diff
@@ -137,6 +140,145 @@ void XCfunctional::initialize(const std::string& input_line, bool polarized,
     }
 }
 
+void XCfunctional::set_external_parameters_if_a_functional_has_been_specified(std::stringstream &line) {
```

Comment:
probably overkuill to have an extra function here, consider to move that check into the "set_external_paratmer" function. 
In the error message: Print example output (as in the sample input file) that is super helpful for users